### PR TITLE
Add support metadata

### PR DESCRIPTION
**Pull request by greg0ire**
This should result in more links available on Packagist.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,6 +22,13 @@
         {"name": "Jonathan Wage", "email": "jonwage@gmail.com"},
         {"name": "Marco Pivetta", "email": "ocramius@gmail.com"}
     ],
+    "support": {
+        "email": "doctrine-user@googlegroups.com",
+        "issues": "https://github.com/doctrine/doctrine2/issues",
+        "source": "https://github.com/doctrine/doctrine2",
```
Comment (greg0ire) [inline composer.json]: Here I hesitated with https://github.com/doctrine/doctrine2/tree/master/lib/Doctrine/ORM, and I notice that Packagist already displays a link pointing to https://github.com/doctrine/doctrine2/tree/master. Tell me what you think is best.

File: composer.json [hunk 2]
```diff
@@ -22,6 +22,13 @@
...
+        "rss": "https://github.com/doctrine/doctrine2/releases.atom",
```
Comment (greg0ire) [inline composer.json]: I wasn't sure what to put here, @Majkl578 what were you thinking about? Something else maybe?

Comment (Majkl578) [reply composer.json]: Should be fine.

File: composer.json (hunk 1 above)
Comment (Majkl578) [reply composer.json]: Leave it like this, otherwise it'd be another maintenance burden.

**Comment by Majkl578** [comment]
Can you also add `docs`?

**Comment by greg0ire** [comment]
Sure thing! I also sorted all this alphabetically.

**Review by Majkl578 — DISMISSED** [review]

File: composer.json [hunk 3]
```diff
@@ -22,6 +22,14 @@
         {"name": "Jonathan Wage", "email": "jonwage@gmail.com"},
         {"name": "Marco Pivetta", "email": "ocramius@gmail.com"}
     ],
+    "support": {
+        "chat": "https://www.doctrine-project.org/slack",
+        "docs": "https://www.doctrine-project.org/projects/orm.html",
+        "email": "doctrine-user@googlegroups.com",
+        "issues": "https://github.com/doctrine/doctrine2/issues",
+        "rss": "https://github.com/doctrine/doctrine2/releases.atom",
+        "source": "https://github.com/doctrine/doctrine2",
```
Comment (lcobucci) [inline composer.json]: Invalid json here, comma not allowed

**Review by lcobucci — CHANGES_REQUESTED** [review]

File: composer.json (hunk 3 above)
Comment (greg0ire) [reply composer.json]: Woops, good catch!

**Review by lcobucci — APPROVED** [review]

**Comment by lcobucci** [comment]
@greg0ire :ship:

**Comment by lcobucci** [comment]
`rss` seems to not be used though :smile: https://packagist.org/packages/doctrine/orm

**Pull request merged**